shardingsphere-sql-parser/shardingsphere-sql-parser-engine/src/main/java/org/apache/shardingsphere/sql/parser/core/database/visitor/SQLVisitorRule.java

```diff
@@ -481,7 +481,7 @@ public enum SQLVisitorRule {
     ALTER_DOMAIN("AlterDomain", SQLStatementType.DDL),
     
     ALTER_POLICY("AlterPolicy", SQLStatementType.DDL),
-    
```

Comment:
Please keep same indent with the previous line.

Reply:
> Please keep same indent with the previous line.

Done.